src/_locales/en/messages.json

```diff
@@ -0,0 +1,23 @@
+{
+  "enabled": {
+    "message": "Enabled"
+  },
+  "comment": {
+    "message": "Comment"
+  },
+  "color": {
+    "message": "color"
+  },
+  "textShadow": {
+    "message": "textShadow"
+  },
+  "rows": {
+    "message": "rows"
+  },
+  "speed": {
+    "message": "speed"
+  },
```

Comment:
We used capitalized values before, therefor if there are no reason to use uncapitalized value, please use capitalized values.

Reply:
Thx review!! fixed it!